app/models/manageiq/providers/redhat/infra_manager/vm/operations/snapshot.rb

```diff
@@ -1,4 +1,18 @@
 module ManageIQ::Providers::Redhat::InfraManager::Vm::Operations::Snapshot
+  extend ActiveSupport::Concern
+
+  included do
+    supports :snapshots do
+      unless ext_management_system&.supports_snapshots?
```

Comment:
@durandom I would think we can move to 2.3 features after euwe is shipped.  Note, we'd have to make sure all dependent gems/providers/etc. are 2.3 compatible and get everybody on ruby 2.3.